Common/src/main/java/dk/sdu/se4/common/entity/parts/TestPart.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.sdu.se4.common.entity.parts;
+
+import dk.sdu.se4.common.data.GameData;
+import dk.sdu.se4.common.data.World;
+import dk.sdu.se4.common.entity.Entity;
+
+/**
+ *
+ * @author steff
+ */
+public class TestPart implements EntityPart{    
```

Comment:
Should move this test part into the test folder.

Reply:
oh Thanks :)
I do :)